### PR TITLE
Point at 3.0-prerelease git tree

### PR DESCRIPTION
**Pull request by inducer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by inducer** [comment]
@conda-forge-admin, please rerender

**Comment by inducer** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,18 @@
 {% set name = "ocl-icd" %}
-{% set version = "2.2.12" %}
-{% set sha256 = "17500e5788304eef5b52dbe784cec197bdae64e05eecf38317840d2d05484272" %}
+{% set version = "2.99_97e3f" %}
```
Comment (inducer) [inline recipe/meta.yaml]: @isuruf What do you think of this? (Particularly the version number? Also, is there a way to make this not the installed-by-default version?) Now that CL 3.0 is released, I'm trying to get support for it in shape. But to do so in PyOpenCL, I need the 3.0 headers.

x-ref: https://github.com/inducer/pyopencl/pull/376

Comment (isuruf) [reply recipe/meta.yaml]: We can publish this recipe to `conda-forge/label/ocl-icd-dev` channel. That way, it's not installed by default and the version can be `3.0.0.dev0` assuming `ocl-icd` is going to release `3.0.0` next.

Comment (inducer) [reply recipe/meta.yaml]: > We can publish this recipe to conda-forge/label/ocl-icd-dev channel.

That sounds like a good plan. How does that work?

> assuming `ocl-icd` is going to release `3.0.0` next.

Hard to know, but likely.

Comment (inducer) [inline recipe/meta.yaml]:
```suggestion
{% set version = "3.0.0.dev0" %}
```

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (inducer) [reply recipe/meta.yaml]: Thanks!

**Pull request merged**